src/src/modules/auth/auth.service.ts

```diff
@@ -493,6 +493,17 @@ export class AuthService {
 			}
 		}
 
+		if (
+			!['beneficiaries', 'facilitators'].includes(group) ||
```

Comment:
can we simpliy this by moving into the if condition that laready exists at top of the method?

```
if (body.role === 'facilitator') {
    // Existing code
    
    // Validate role specific fields
    if (!body.role_fields.parent_ip) {
        let misssingFieldsFlag = true;
    }
}
else {
   // 
   if (!body.role_fields.facilitator_id) {
           let misssingFieldsFlag = true;
   }
}

if (misssingFieldsFlag) {
  throw new BadRequestException({
    success: false,
    message: 'Invalid parameters',
  });
```
}

Reply:
Done